.storybook/main.ts

```diff
@@ -70,4 +73,47 @@ module.exports = {
       plugins: [],
     };
   },
+  webpackFinal(config, { configType }) {
+    if (configType === 'DEVELOPMENT') {
+      updateCSSLoaderPlugin(config);
+    }
+    return config;
+  },
 };
+
+/**
+ * Updates the `css-loader` webpack plugin to make class names human readable.
+ *
+ * NOTE: This should only be used for local development.
+ */
+function updateCSSLoaderPlugin(config: Configuration): Configuration {
+  config.module?.rules?.forEach((rule) => {
+    if (rule && typeof rule === 'object' && Array.isArray(rule.use)) {
+      const isRuleForCSS = rule.test?.toString() === '/\\.css$/';
+      if (isRuleForCSS) {
+        rule.use.forEach((ruleSetRule) => {
+          if (
+            typeof ruleSetRule === 'object' &&
+            ruleSetRule?.loader?.includes('node_modules/css-loader')
+          ) {
+            ruleSetRule.options = {
+              // @ts-expect-error css-loader doesn't accept "string" options
+              // and will either be an object or undefined
+              ...ruleSetRule.options,
+              modules: {
+                // @ts-expect-error css-loader doesn't accept "string" options
+                // and will either be an object or undefined
+                ...ruleSetRule.options?.modules,
+                localIdentName: '[name]__[local]',
```

Comment:
I wonder if we should either:
- Keep the hash at the end of the name (e.g. [name]_[local]_[hash])
- Or include the path (e.g. [path]_[name]_[local])

That way the classes are a bit more guaranteed to be unique, like they will be in production.

Probably low chance of conflict as-is, but it could be nice to make double extra sure.

Reply:
the old behavior before the storybook 7 upgrade [included the hash at the end](https://61313967cde49b003ae2a860-lwbwxnvynb.chromatic.com/?path=/story/components-accordion--default) so we could go back to that

Reply:
I'll add the `hash`. I think adding the `path` is too verbose for class names given the depth and structure of some directories

Reply:
Classes with [hash]

<img width="490" alt="image" src="https://github.com/chanzuckerberg/edu-design-system/assets/94476616/f2377f9c-e7d5-49c1-a42f-f37a4370541d">